.scalafmt.conf

```diff
@@ -1,21 +1,52 @@
-version = 2.1.0
-
-style = defaultWithAlign
-
-docstrings                 = JavaDoc
-indentOperator             = spray
-maxColumn                  = 120
-lineEndings                = preserve
-rewrite.rules              = [RedundantParens, SortImports, AvoidInfix]
-unindentTopLevelOperators  = true
-align.tokens               = [{code = "=>", owner = "Case"}]
-align.openParenDefnSite    = false
-align.openParenCallSite    = false
-optIn.breakChainOnFirstMethodDot = false
-optIn.configStyleArguments = false
-danglingParentheses = false
-spaces.inImportCurlyBraces = true
-rewrite.neverInfix.excludeFilters = [
+version                                  = 3.6.1
+runner.dialect                           = scala213
+project.git                              = true
+style                                    = defaultWithAlign
+docstrings.style                         = Asterisk
+docstrings.wrap                          = false
+indentOperator.preset                    = spray
+maxColumn                                = 120
```

Comment:
With todays wide/ultra-wide monitors I think we could keep this at 180 (old setting didn't seem to apply that strictly)

Reply:
Do you feel strongly about this? From what I have seem most modern Scala OS code uses 120, not 180 and I would like to keep the formatting as standard as possible.

Reply:
Not strongly, just saw some occurrence where the re-formatting split things into a new line I didn't necessarily think improved readability.

Reply:
I'm not working on a ultra-wide monitor, I work on a 16-inch MacBook, and before that, I worked on a 13-inch MacBook. I also keep my font at 16, because when reading code, the presentation quality is better than quantity.  180 chars would not fit on any screen I worked on in the past 10 years 🙂

Also, I'd argue that having to move your eyes from left to right for 180 chars can be terrible for readability. The consensus is that continuous lines have to be under 80 chars for the best results, a claim somewhat corroborated by at least one study I saw. Of course, in programming we also have indentation, but if you have that much indentation to need more than 120 chars, maybe that code is a little too complex in terms of branching.

If you have a wide-monitor, a better use of that space is to open 2 files side by side (e.g., in diffs), which is also good for PR reviews.

Personally I'd go with 100. The only reasons for why I don't like that is because Scala's function signatures can get awkwardly long, and formatting those signatures becomes a chore. I'd wish, however, for Scalafmt to have different settings depending on context — e.g., leave the function signatures on one line, but anything after the `=` hard-wrap it at 100.

Going with something more than 120, for me, it's like not having any limits at all. Which is fine, but then I personally complain about code not being readable in code reviews, and we desperately want to avoid that. Which is the value that Scalafmt brings.

Reply:
I second @alexandru point. 

My preferred coding layout is 2 pane side by side, so a maxColumn of 180 would not fit even on my widescreen monitor (in VSCode I get about 160 chars per pane). 

Furthermore my brain tends to parse better shorter multi-line expressions than longer single line ones, so personally I'd stick with 120.

Reply:
So judging by the conversation here I would just leave it at 120, this is what the community generally converges too.

Reply:
I've also switched from 120 to 100 over the past decade for side-by-side diff.

But if it was 120, probably no need to make the formatting diff go all crazy.

Reply:
Yeah setting it to 120 would explode the diff and honestly in a way where its no longer trivial to compare it visually. At least right now if there is a difference its easy to make out.

Reply:
On final thought I will resolve this conversation, changing the value from 120 is massively detrimental to the PR's goals of trying to minimize the diff.

If we are going to change this, it would be on the alternative approach where instead of minimizing the diff we instead just try to use a more standard scalafmt configuration accepting that the diff would be massive, i.e. https://github.com/mdedetrich/akka-apache/pull/2#issuecomment-1291044281 .

If people want to entertain this then we can do a github discussions poll (once its enabled) to gauge whether the community at large would prefer that and then one can always create a formal apache vote for it.